docs/pages/core/quickstart.mdx

```diff
@@ -31,9 +31,15 @@ npm i -s @account-kit/core @account-kit/infra
 
 </CodeBlocks>
 
-## Get your API keys
+## Get your Environment Variables
 
-<Markdown src="../../shared/get-api-key.mdx" />
+1. Create an app in the [dashboard](https://dashboard.alchemy.com/signup) and copy the **API Key**.
+
+2. Create a **`configuration`** in the [`Smart Wallets` dashboard](https://dashboard.alchemy.com/services/smart-wallets/configuration) to enable login methods.
+
+3. Create a policy in your [gas manager dashboard](https://dashboard.alchemy.com/services/gas-manager/configuration) to set rules for sponsorship.
```

Comment:
This seems like a pretty common pattern for new setups. I wonder if it's worth writing up an isolated and reusable guide for this as we have similar material that's scattered around docs. Not blocking but just food for thought

Reply:
yeah agreed - the problem currently is alot of them are context dependent:

- some will explicitly say to store the `.env` variables (because we know what they are)
- others cant use `.env` variables (like react native)
- react native guides "from scratch" are super long and dont mention gas policy currently (its a next step) so we wouldnt want that one in there
- this one suggests using `.env` variables, but the framework is unknown.

In the past we've created shared snippets to our detriment because someone updates it without considering how it affects all the other places. I'm trying to avoid that more at the moment

Reply:
yup, good point - will take a look at this separately - theres a dedicated asana task for this from ava